nixops/deployment.py

```diff
@@ -139,7 +142,7 @@ def machines(self) -> Dict[str, nixops.backends.MachineState]:
         }
 
     @property
-    def active(
+    def active_machines(
         self,
     ) -> Dict[str, nixops.backends.MachineState]:  # FIXME: rename to "active_machines"
```

Comment:
Let's mark that FIXME as a FIXEDYOU, and move this rename and its fallout in to a separate commit near the beginning of the series?